### DISCUSSION TITLE
Proposta fatta circa un anno fa

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#348] [Idee e proposte]
Buonasera,circa un anno fa ,avevamo riscontrato diversi problemi con i genitori che venivano a lamentarsi delle valutazioni date a fine anno /fine trimestre in pagella ai loro figli ,così vi abbiamo chiesto di aggiungere un tasto con scritta "presa visione" vicino ad ogni singolo voto che giornalmente i docenti mettono ,in modo che ,se capitano più volte questi episodi di lamentele da parte dei genitori il docente poteva vedere nel registro che il voto è scaturito dalla media generale dei singoli voti di cui i genitori non hanno mai preso visione.Poi abbiamo chiesto anche di far apparire nel portale dei genitori/alunni il nome del docente che inserisce la lezione in modo da avere famigliarità e riuscire ad attribuire ad ogni materia il docente.Poi così come è diviso nel portale genitori/alunni "argomento svolto " e "attività da svolgere",sarebbe bello vedere la stessa cosa dal lato docente,che invece mette se non sbaglio argomento/attività non facendo evincere molto

**Comment by trinko** [comment]
La segno come richiesta.
